psydac/polar/dense.py

```diff
@@ -252,6 +256,10 @@ def __isub__( self, v ):
         self._data -= v._data
         return self
 
+    # ...
+    def update_ghost_regions( self, *, direction=None ):
+        pass
+
```

Comment:
Same comment as above. Please move this method to the section below.

Reply:
done